### DISCUSSION TITLE
Welcome to cinemax Discussions!

### DISCUSSION THREAD

**Discussion by BeamlakAschalew** [BeamlakAschalew/flixquest#48] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BeamlakAschalew** [comment]
Update to fix streaming issue is in the making.

**Reply by MrConqueror** [reply]
Bruh you know really need to build Built-in Player

**Reply by BeamlakAschalew** [reply]
We have that now, infact it's the only way now. Maybe if we're thinking of different things, what's on your mind?

**Reply by MrConqueror** [reply]
that's it? that's your built-in Player looks like web Player

**Reply by BeamlakAschalew** [reply]
> that's it? that's your built-in Player looks like web Player

Yeah it looks like that but I'm constantly looking for better video players, I don't have the expertise to develop one.
